lib/menu.rb

```diff
@@ -0,0 +1,54 @@
+require 'csv'
+
+
+class Menu
+  
+  attr_reader :menu     
```

Comment:
Don't expose the internal hash of objects.  And don't call it `menu` as it's now unclear what a menu is.  Is it an internal hash of items, or is it the object that contains an internal hash of items?